ext/js/display/display.js

```diff
@@ -1435,16 +1436,31 @@ export class Display extends EventDispatcher {
             this._dictionaryEntryNodes.push(entry);
             this._addEntryEventListeners(entry);
             this._triggerContentUpdateEntry(dictionaryEntry, entry, i);
+            safePerformance.mark('display:waitMedia:start');
+            safePerformance.mark('display:waitMedia:end');
+            safePerformance.measure('display:waitMedia', 'display:waitMedia:start', 'display:waitMedia:end');
```

Comment:
What is the point of these performance marks and measure? It looks like it's measuring nothing.